serverless/pytorch/foolwood/siammask/nuclio/function.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - kind: WORKDIR
           value: /opt/nuclio
         - kind: RUN
-          value: conda create -y -n siammask python=3.6
+          value: conda create -y -n siammask python=3.7
```

Comment:
Not 3.9?

Reply:
3.7 because Siamak does not support python3.9. Here the function runs inside the conda environment and it's work okay

Reply:
> 3.7 because Siamak does not support python3.9.

Could you provide more details?

Reply:
BTW, this is concerning, because Python 3.7 is already EOL, and 3.8 will reach EOL in 6 months: <https://endoflife.date/python>.

Reply:
> Updating packages leads to errors in the function code.

What are the errors?

Reply:
While I was trying to reproduce the error,I was able to make it so that the function work on python3.9